Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 - Fixed memory allocation on every frame when using UIDocument without EventSystem. [ISXB-953](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-953)
 - Fixed Action Maps name edition which could be inconsistent in Input Action Editor UI.
+- Fixed an update loop in the asset editor.
```

Comment:
Would suggest describing the symptom and the actual issue happening that was fixed here instead or what do you think?